src/Utopia/Messaging/Messages/Email.php

```diff
@@ -24,49 +24,31 @@ public function __construct(
     ) {
     }
 
-    /**
```

Comment:
Why did you remove a bunch of comments in unrelated files?

Reply:
I ran the **composer lint** command and then it removed the comments to resolve the linting errors issue. @gewenyu99

Reply:
????? Really

Reply:
> I ran the **composer lint** command and then it removed the comments to resolve the linting errors issue. 

That doesn't quite make sense...`composer lint` only checks for lint errors. it shouldn't be modifying anything.

Reply:
![lint error](https://github.com/utopia-php/messaging/assets/64208057/e5bc6b07-87b1-4ddf-ac52-4544d91d14de)
@stnguyen90 Sorry, it was **composer format** not composer lint. **composer format** removed those comments because it was causing lint error. However, my files are not causing any lint error.